TODO.txt

```diff
@@ -0,0 +1,7 @@
+- multi-language support during OCR;
```

Comment:
These are all interesting ideas but a .txt file in the root of the repo is not the best place for them. Please remove this file and open an issues (one for every proposal) against this repository.